src/Console/Locale/CopyLocaleCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Console\Locale;
+
+use Backend\Core\Language\Locale;
+use Common\Locale as CommonLocale;
+use Common\ModulesSettings;
+use ForkCMS\Bundle\CoreBundle\Manager\CopyModulesToOtherLocaleManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * This command will copy all the content from all the modules
+ */
+class CopyLocaleCommand extends Command
+{
+    /**
+     * @var CopyModulesToOtherLocaleManager
+     */
+    private $copyModulesToOtherLocaleManager;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $formatter;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
```

Comment:
I don't like abbreviations in code.

Also: why do we have a formatter and an io which is basically the same object?

Reply:
Indeed, removed this variable entirely since we already are using this class as `private $formatter`